hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleHederaStateTest.java

```diff
@@ -449,46 +449,30 @@ void knownServiceNameUsingReadableStates() {
             // Then query it, we find the data we expected to find
             assertThat(states).isNotNull();
             assertThat(states.isEmpty()).isFalse();
-            assertThat(states.size()).isEqualTo(4); // animal and fruit and country
+            assertThat(states.size()).isEqualTo(4); // animal and fruit and country and steam
 
-            final ReadableKVState<String, String> fruitStates = states.get(FRUIT_STATE_KEY);
-            assertThat(fruitStates).isNotNull();
-            assertThat(fruitStates.get(A_KEY)).isSameAs(APPLE);
-            assertThat(fruitStates.get(B_KEY)).isSameAs(BANANA);
-            assertThat(fruitStates.get(C_KEY)).isNull();
-            assertThat(fruitStates.get(D_KEY)).isNull();
-            assertThat(fruitStates.get(E_KEY)).isNull();
-            assertThat(fruitStates.get(F_KEY)).isNull();
-            assertThat(fruitStates.get(G_KEY)).isNull();
-
-            final ReadableKVState<String, String> animalStates = states.get(ANIMAL_STATE_KEY);
-            assertThat(animalStates).isNotNull();
-            assertThat(animalStates.get(A_KEY)).isNull();
-            assertThat(animalStates.get(B_KEY)).isNull();
-            assertThat(animalStates.get(C_KEY)).isSameAs(CUTTLEFISH);
-            assertThat(animalStates.get(D_KEY)).isSameAs(DOG);
-            assertThat(animalStates.get(E_KEY)).isNull();
-            assertThat(animalStates.get(F_KEY)).isSameAs(FOX);
-            assertThat(animalStates.get(G_KEY)).isNull();
+            final ReadableKVState<String, String> fruitState = states.get(FRUIT_STATE_KEY);
+            assertFruitState(fruitState);
```

Comment:
Nit: I think this complaint is questionable. What does it mean? You are testing too much? 😂 The method does not make this test easier. Maybe defining a `Condition<ReadableKVState>` would be an option. I did something similar in `SavepointStackImplTest`. Should not block this PR though.

Reply:
Thanks, will check `SavepointStackImplTest` and will take note to update this in some future PR.